src/backend/routers/iep.test.ts

```diff
@@ -70,7 +85,7 @@ test("basic flow - add/get goals, subgoals, tasks", async (t) => {
   const gotSubgoals = await trpc.iep.getSubgoals.query({
     goal_id: goal1!.goal_id,
   });
-  t.is(gotSubgoals.length, 2);
+  t.is(gotSubgoals.length, 3);
```

Comment:
Could you please explain why this was updated? Thanks!

Reply:
Previously, one of the subgoals was created, and used twice in the test to create a task with the same para. With the addition of the new database constraint, the creation of the second task failed. So I made a new subgoal in order for both tasks to be able to be created successfully, and thus the number of subgoals for that goal is now 3 instead of 2.